lib/text_parser_override.rb

```diff
@@ -0,0 +1,11 @@
+StartingBlocks::TextParser.class_eval do
```

Comment:
This is a personal opinion, but if you are going to override a class, I think it's clearer to use ```class_eval```.

it sticks out more than just declaring the class again.  It also will throw an error if, heaven forbid, the order in which the files are loaded change.